src/tiled/gotodialog.cpp

```diff
@@ -0,0 +1,94 @@
+#include "gotodialog.h"
+
+#include <QApplication>
+#include <QHBoxLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QPushButton>
+#include <QIntValidator>
+#include <QString>
+#include "documentmanager.h"
+#include "mapview.h"
+#include "mapscene.h"
+#include "mapreader.h"
+#include "mapdocument.h"
+#include "maprenderer.h"
+
+using namespace Tiled::Internal;
+using namespace Tiled;
+
+GotoDialog *GotoDialog::mInstance;
+
+GotoDialog::GotoDialog(QWidget *parent, Qt::WindowFlags f )
```

Comment:
Yeah, I think if we add the goto object it might be better to call it FindDialog